utils/training.py

```diff
@@ -54,7 +54,10 @@ def evaluate(model: ContinualModel, dataset: ContinualDataset, last=False) -> Tu
         correct, correct_mask_classes, total = 0.0, 0.0, 0.0
         for data in test_loader:
             with torch.no_grad():
-                inputs, labels = data
+                if dataset.HAS_LABEL_DRIFT:
+                    inputs, labels, _ = data
```

Comment:
instead of if you can write:
inputs = data[0]
labels = data[1]

and later
not_aug_inputs = data[2]
logits = data[-1]